mtmount/Dockerfile

```diff
@@ -0,0 +1,115 @@
+# This docker file cannot fully automate the EUI installation. You must carry on
+# with the installation of the VIPM package installs to complete the installation. 
+
+FROM centos:centos7
+
+LABEL maintainer Andrew Heyer <aheyer@lsst.org>
+
+ENV USER ${USER:-saluser}
+ENV HOME /home/saluser
+USER root
+# Create user and group
+RUN groupadd lsst && adduser -u 1004 -m -g lsst -s /bin/bash saluser
+
+COPY lsst-ts-nexus.repo /etc/yum.repos.d/lsst-ts-nexus.repo
+COPY wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
+RUN rpm --import http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco
+RUN yum install -y git wget
+RUN yum install -y OpenSpliceDDS-6.9.0
+
+#Install and update various packages needed for Labview, VPM, and be able to
+#compile C++ tma management 
+#RUN yum update - for some reason, adding yum updated fails to work, this may
+# need attention in the future.
+RUN yum groupinstall -y "Development Tools"
+RUN yum install -y cmake boost-devel.x86_64
+
+RUN yum install -y epel-release
+RUN yum install -y geany java-1.7.0-openjdk-devel
+RUN yum install -y glibc.i686 libstdc++.so.6 libXinerama.i686 && \
```

Comment:
Compressing those lines into one long yum install -y .... \ ... will make it faster with smaller container image